README.md

```diff
@@ -73,33 +100,45 @@ curation_pipeline = Sequential([
 curated_dataset = curation_pipeline(dataset)
 ```
 
-NeMo Curator provides a collection of robust python modules that can be chained together to construct your entire data curation pipeline. These modules can be run on your local machine or in a distributed compute environment like SLURM with no modifications. NeMo Curator provides simple base classes that you can inherit from to create your own filters, document modifiers, and other extensions without needing to worry about how they scale. The [examples](examples/) directory contains a bunch of scripts showcasing each of these modules. The data curation section of the [NeMo framework user guide](https://docs.nvidia.com/nemo-framework/user-guide/latest/datacuration/index.html) provides in-depth documentation on how each of the modules work. If you need more information to modify the NeMo Curator for your usecase, the [implementation section](#implementation) provides a good starting point.
+### Explore NeMo Curator Tutorials
 
-### Scripts
+To get started with NeMo Curator, you can follow the tutorials available here: [Tutorials]
+(https://github.com/NVIDIA/NeMo-Curator/tree/main/tutorials). These tutorials include:
 
-We provide CLI scripts to use as well in case those are more convienent. The scripts under `nemo_curator/scripts` map closely with each of the created python modules. Visit the [documentation](docs) for each of the python modules for more information about the scripts associated with it.
++- tinystories which focuses on data curation for training from scratch.
```

Comment:
Wrap in backticks so it appears as `tinystories`

Reply:
Added backticks